sdks/python/apache_beam/testing/test_stream.py

```diff
@@ -276,8 +295,11 @@ def to_runner_api_parameter(self, context):
   @PTransform.register_urn(
       common_urns.primitives.TEST_STREAM.urn,
       beam_runner_api_pb2.TestStreamPayload)
-  def from_runner_api_parameter(payload, context):
+  def from_runner_api_parameter(ptransform, payload, context):
     coder = context.coders.get_by_id(payload.coder_id)
+    output_tags = set(
+        None if k == 'None' else k for k in ptransform.outputs.keys())
```

Comment:
Following up my other comment, what happens if the user tags their output to be the string 'None'?

Reply:
This behavior isn't unique in the TestStream. This is consistent with the rest of the Python SDK with handling PCollection tags being None. As for what happens, I don't know, there are probably many subtle things that may go wrong.